_data/kathrynparadis/kathrynparadis.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Kathryn Paradis
+institution: University of Hawaii at Manoa
+profile_pic: kat.jpg
+quote: "She believed she could, so she did -R.S. Grey"
```

Comment:
Hey @kathrynparadis , could you avoid using ( " ) in the quote to keep the formatting intact ?

Reply:
Hey @monsij, thank you for reviewing my request. I've removed the quotations, were there any other errors?